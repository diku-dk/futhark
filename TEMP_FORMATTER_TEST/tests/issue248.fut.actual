c235ba4ccc5e1df3eba4a416ba0dc722
