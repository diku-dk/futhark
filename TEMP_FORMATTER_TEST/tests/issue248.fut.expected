94dadcaf5a5d7649355caaa1334520fc
